src/selenium.ts

```diff
@@ -76,6 +76,7 @@ async function seleniumStart(
 
   const config: selenium.StartOpts = SELENIUM_OVERRIDES || {};
   config.seleniumArgs = ['-port', port.toString()].concat(opts.args);
+  config.seleniumJavaArgs = [];
```

Comment:
Here you called it `seleniumJavaArgs` and not just `javaArgs`?

Reply:
Oh woops.

Reply:
i fixed that :)